meson_options.txt

```diff
@@ -6,7 +6,7 @@ option('engines',
 
 option('loaders',
    type: 'array',
-   choices: ['', 'svg'],
+   choices: ['', 'svg', 'png'],
    value: ['svg'],
    description: 'Enable Vector File Loader in thorvg')
```

Comment:
since we have here now svg and png I'd change the description

Reply:
@mgrudzinska updated.